lib/npq_api_endpoint_route.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class NpqApiEndpointRoute
+  def self.matches?(_request)
+    !FeatureFlag.active?(:disable_npq_endpoints)
```

Comment:
I wonder if we should change this to check the ENV is separation instead and switch it to the feature flag later to avoid hitting the db everytime those URLs are hit? let's discuss